pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -123,7 +124,8 @@ func Pod(source *v1alpha1.CatalogSource, name string, image string, labels map[s
 					LivenessProbe: &v1.Probe{
 						Handler: v1.Handler{
 							Exec: &v1.ExecAction{
-								Command: []string{"grpc_health_probe", "-addr=localhost:50051"},
+								Command: []string{"grpc_health_probe", "-addr=localhost:50051",
```

Comment:
The default value for liveness probe timeout is [1 second](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes), so we probably want to set the `TimeoutSeconds` field to 10 seconds as well.

Reply:
yes, I agree.